manifests/j/jj-vcs/jj/0.24.0/jj-vcs.jj.installer.yaml

```diff
@@ -13,5 +16,4 @@ Installers:
   InstallerUrl: https://github.com/jj-vcs/jj/releases/download/v0.24.0/jj-v0.24.0-x86_64-pc-windows-msvc.zip
   InstallerSha256: 76AAE7589FCC32777DB3915A2C26EDC1EE02F1A530DF97B1E8552DC6C7D6E7E2
 ManifestType: installer
-ManifestVersion: 1.9.0
-ReleaseDate: 2024-12-04
+ManifestVersion: 1.10.0

```

Comment:
Looks like release date was removed. Not sure if this is correct or not.

Reply:
Reverted